src/term.js

```diff
@@ -2511,6 +2523,12 @@ Terminal.prototype.keyDown = function(ev) {
         key = '\x1b[6~';
       }
       break;
+    // CTRL-K
+    case 75:
+      if ((!isMac && ev.ctrlKey) || (isMac && ev.metaKey)) {
+        this.clear();
+        return cancel(ev);
+      }
```

Comment:
Is this fall-through intentional? Now typing the letter K will trigger an F1 key?